examples/standalone-controls.html

```diff
@@ -11,39 +12,38 @@
+  </media-controller>
 
-<h1>Standalone Controls Example</h1>
-<p>This example shows how individual controls can be used without the <code>media-container</code> element.</p>
+  <h2>Outside the control-bar element</h2>
+  <media-play-button media-controller="controller"></media-play-button>
+  <media-mute-button media-controller="controller"></media-mute-button>
+  <media-seek-backward-button media-controller="controller"></media-seek-backward-button>
+  <media-seek-forward-button media-controller="controller"></media-seek-forward-button>
+  <media-volume-range media-controller="controller"></media-volume-range>
+  <media-time-range media-controller="controller"></media-time-range>
+  <media-playback-rate-button media-controller="controller"></media-playback-rate-button>
+  <media-pip-button media-controller="controller"></media-pip-button>
+  <media-fullscreen-button media-controller="controller"></media-fullscreen-button>
 
-<media-controller id="controller">
-  <video slot="media" src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"></video>
-</media-controller>
-
-<h2>Outside the control-bar element</h2>
-<media-play-button media-controller="controller"></media-play-button>
-<media-mute-button media-controller="controller"></media-mute-button>
-<media-seek-backward-button media-controller="controller"></media-seek-backward-button>
-<media-seek-forward-button media-controller="controller"></media-seek-forward-button>
-<media-volume-range media-controller="controller"></media-volume-range>
-<media-time-range media-controller="controller"></media-time-range>
-<media-playback-rate-button media-controller="controller"></media-playback-rate-button>
-<media-pip-button media-controller="controller"></media-pip-button>
-<media-fullscreen-button media-controller="controller"></media-fullscreen-button>
-
-<h2>Inside the control-bar element</h2>
-<p>Note: You only need to set the <code>media-controller=</code> attribute on the <code>media-control-bar</code>. The media-controller will pass the media state details to their child media-chrome elements automatically.</p>
-
-<media-control-bar media-controller="controller">
-  <media-play-button></media-play-button>
-  <media-mute-button></media-mute-button>
-  <media-seek-backward-button></media-seek-backward-button>
-  <media-seek-forward-button></media-seek-forward-button>
-  <media-volume-range></media-volume-range>
-  <media-time-range></media-time-range>
-  <media-playback-rate-button></media-playback-rate-button>
-  <media-pip-button></media-pip-button>
-  <media-fullscreen-button></media-fullscreen-button>
-</media-control-bar>
+  <h2>Inside the control-bar element</h2>
+  <p>Note: You only need to set the <code>media-controller=</code> attribute on the <code>media-control-bar</code>. The media-controller will pass the media state details to their child media-chrome elements automatically.</p>
```

Comment:
Good catch. I don't believe it is. I'll create an issue and make this work with the new architecture.
Proposal for you: 
1. instead of assuming an `id` (which has to be globally unique on the DOM), we support any query selector. 
2. If there are concerns for backwards compatibility, we can do (1) as a separate attribute (`media-controller-selector`?)